models/common.py

```diff
@@ -123,7 +121,7 @@ def __init__(self, c1, c2, n=1, shortcut=True, g=1, e=0.5):  # ch_in, ch_out, nu
         self.cv3 = nn.Conv2d(c_, c_, 1, 1, bias=False)
         self.cv4 = Conv(2 * c_, c2, 1, 1)
         self.bn = nn.BatchNorm2d(2 * c_)  # applied to cat(cv2, cv3)
-        self.act = nn.LeakyReLU() #nn.SiLU()
+        self.act = nn.LeakyReLU()
```

Comment:
what's the change here?

Reply:
removed the comment #nn.SiLU. Which was the previously used activation function

Reply:
removed unwanted comment